gc3libs/backends/shellcmd.py

```diff
@@ -713,16 +713,15 @@ def spooldir(self, value):
     def _init_spooldir(self):
         """Set `self.spooldir` to a sensible value."""
         rc, stdout, stderr = self.transport.execute_command(
-            'cd "${TMPDIR:-/var/tmp}" && pwd')
+            'cd "{0}" && pwd'.format(gc3libs.Default.SPOOLDIR))
```

Comment:
See similar comment above: `self._init_spooldir` is useless with the new static default.